<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signUp</class>
 <widget class="QDialog" name="signUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/Resources/logInBackground.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="blackLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>451</width>
     <height>371</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0d0c0c;
border-radius: 40px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white</string>
   </property>
   <property name="text">
    <string>HOME MANAGEMENT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 2px solid gray;
border-radius: 10px;
padding: 0 5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>USERNAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 2px solid gray;
border-radius: 10px;
padding: 0 5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>EMAIL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 2px solid gray;
border-radius: 10px;
padding: 0 5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 2px solid gray;
border-radius: 10px;
padding: 0 5px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>REPEAT PASSWORD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">button-&gt;setStyleSheet(
    &quot;QPushButton {&quot;
    &quot;   background-color: white;&quot;
    &quot;   color: black;&quot;
    &quot;   border-radius: 10px;&quot;
    &quot;   border: none;&quot;
    &quot;   padding: 10px;&quot;
    &quot;}&quot;
    &quot;QPushButton:hover {&quot;
    &quot;   background-color: rgba(255, 255, 255, 0.9);&quot;
    &quot;}&quot;
    &quot;QPushButton:pressed {&quot;
    &quot;   background-color: rgba(255, 255, 255, 0.8);&quot; // Lekkie przyciemnienie po naciśnięciu
    &quot;}&quot;
);</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logInButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: white;
</string>
   </property>
   <property name="text">
    <string>Already a user? Log in!</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="budgetSpinBox">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>250</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    border: 2px solid gray;
    border-radius: 10px;
    padding-right: 15px; /* Zostaw miejsce na przyciski góra/dół */
    padding-left: 5px;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="suffix">
    <string> zł</string>
   </property>
   <property name="maximum">
    <double>100000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
